### DISCUSSION TITLE
Do you know what are (x:4)(x:5)(x:2) numbers while debugging in my React-XState finite-state machines? Image attached.

### DISCUSSION THREAD

**Discussion by nicoPPLSI** [statelyai/xstate#3961] [General]
what are (x:4)(x:5)(x:2)?
https://prnt.sc/8tGM0PWhCsqR
![img1](https://user-images.githubusercontent.com/108067329/230960219-d1b6a28e-c7e5-49a1-8c3c-8995c314e1d3.png)

**Comment by davidkpiano** [comment]
Yes, these are called "session IDs", which are the unique identifiers for these actors.

**Reply by nicoPPLSI** [reply]
thanks! Does it mean that there are two instances of the "orchestration" machine running? 
I'm wondering why there are two "orchestration" machines in memory 🤔?
Where can I get more information about it?

https://xstate.js.org/docs/packages/xstate-inspect/#implementing is is everything I could find.

thanks!
